src/Components/App.tsx

```diff
@@ -1,5 +1,6 @@
 import React, { Component } from 'react';
 import '../Styles/App.css';
+import Breweries from './Breweries';
 
 // state is going to have one key which is breweries
```

Comment:
I love having this to explain the code! Just need to take it out for the final version.